mlflow/sklearn/__init__.py

```diff
@@ -958,6 +1422,21 @@ def patched_fit(original, self, *args, **kwargs):
                     FLAVOR_NAME, class_def, func_name, patched_fit, manage_run=True,
                 )
 
+        for func_name in ["predict", "predict_proba"]:
```

Comment:
I remove patching for `model.transform` because I cannot find out a case, which metric API use transformed result (returned by `model.transform`) as an input argument.

Reply:
This is estimators which include `transform`:

<details>
  <summary>list</summary>
AdditiveChi2Sampler
BernoulliRBM
Binarizer
Birch
CCA
ColumnTransformer
CountVectorizer
DictVectorizer
DictionaryLearning
FactorAnalysis
FastICA
FeatureAgglomeration
FeatureHasher
FeatureUnion
FunctionTransformer
GaussianRandomProjection
GenericUnivariateSelect
GridSearchCV
HalvingGridSearchCV
HalvingRandomSearchCV
HashingVectorizer
IncrementalPCA
Isomap
IsotonicRegression
IterativeImputer
KBinsDiscretizer
KMeans
KNNImputer
KNeighborsTransformer
KernelCenterer
KernelPCA
LabelBinarizer
LabelEncoder
LatentDirichletAllocation
LinearDiscriminantAnalysis
LocallyLinearEmbedding
MaxAbsScaler
MinMaxScaler
MiniBatchDictionaryLearning
MiniBatchKMeans
MiniBatchSparsePCA
MissingIndicator
MultiLabelBinarizer
NMF
NeighborhoodComponentsAnalysis
Normalizer
Nystroem
OneHotEncoder
OrdinalEncoder
PCA
PLSCanonical
PLSRegression
PLSSVD
PatchExtractor
Pipeline
PolynomialCountSketch
PolynomialFeatures
PowerTransformer
QuantileTransformer
RBFSampler
RFE
RFECV
RadiusNeighborsTransformer
RandomTreesEmbedding
RandomizedSearchCV
RobustScaler
SelectFdr
SelectFpr
SelectFromModel
SelectFwe
SelectKBest
SelectPercentile
SequentialFeatureSelector
SimpleImputer
SkewedChi2Sampler
SparseCoder
SparsePCA
SparseRandomProjection
StackingClassifier
StackingRegressor
StandardScaler
TfidfTransformer
TfidfVectorizer
TruncatedSVD
VarianceThreshold
VotingClassifier
VotingRegressor
<details>

Reply:
Does it hurt to include these? Is there any extra maintenance cost? This is a lot of estimators to exclude.

Reply:
No hurt. I can add back transform patching if you prefer it.

Reply:
added back.